wandb/sdk/launch/registry/google_artifact_registry.py

```diff
@@ -182,22 +182,19 @@ def check_image_exists(self, image_uri: str) -> bool:
         _logger.info(
             f"Checking if image {image_uri} exists. In Google Artifact Registry {self.uri}."
         )
-
-        return False
-        # TODO: Test GCP Artifact Registry image exists to get working
-        # repo_uri, _ = image_uri.split(":")
-        # if repo_uri != self.get_repo_uri():
-        #     raise LaunchError(
-        #         f"The image {image_uri} does not belong to the Google Artifact "
-        #         f"Repository {self.get_repo_uri()}."
-        #     )
-        # credentials = self.environment.get_credentials()
-        # request = google.cloud.artifactregistry.GetTagRequest(parent=image_uri)
-        # client = google.cloud.artifactregistry.ArtifactRegistryClient(
-        #     credentials=credentials
-        # )
-        # try:
-        #     client.get_tag(request=request)
-        #     return True
-        # except google.api_core.exceptions.NotFound:
-        #     return False
+        repo_uri, _ = image_uri.split(":")
```

Comment:
Woah, did this all just work?

Reply:
No sorry I actually didn't mean to include this in this PR. But I'm just gonna fix it rn